sdk/resourcemanager/Azure.ResourceManager.Core/tests/Scenario/PreDefinedTagsOperationsTests.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.Core.Tests
+{
+    public class PreDefinedTagsOperationsTests : ResourceManagerTestBase
```

Comment:
validate code coverage of
PredefinedTagsOperations
PredefinedTagsContainer
TagsRestOperations
TagsOperations
are all above 80%

Reply:
Done